sign/options.go

```diff
@@ -43,6 +43,7 @@ type Options struct {
 	// is provided (i.e. it's not used for keyless signing).
 	// Defaults to a function that reads from stdin and asks for confirmation
 	PassFunc cosign.PassFunc
+	CertPath string
```

Comment:
Where is this used? If I'm not mistaken, it seems like it's not used at all.

Reply:
Removing